File: LaddersAndSlides_GameEngine/Engine/GameStateEngine.cs
namespace LaddersAndSlidesW8UI.Processing
{
    public enum GameStateEngine
    {
        InitialGameState,
        ArrowEvent,
        ArrowDelayedEvent,
        PlayerEvent,
        TurnComplete,
        PlayerSpecialMoveTransportCalculateEvent,
        GetNextPlayer,
        PlayerSpecialMoveTransportMoveEvent
    }
}

File: src/engine/GameStateEngine.ts
export enum GameStateEngine {
  InitialGameState = 'InitialGameState',
  ArrowEvent = 'ArrowEvent',
  ArrowDelayedEvent = 'ArrowDelayedEvent',
  PlayerEvent = 'PlayerEvent',
  TurnComplete = 'TurnComplete',
  GetNextPlayer = 'GetNextPlayer',
  PlayerSpecialMoveTransportMoveEvent = 'PlayerSpecialMoveTransportMoveEvent',
  WinnerDeclared = 'WinnerDeclared'
}
